Modules/Generators/PackageGenerator/Sources/PackageGenerator/PackageGenerator.swift

```diff
@@ -72,7 +72,14 @@ public struct PackageGenerator: PackageGeneratorProtocol {
         let path = try createFolderIfNecessary(folder: "Sources", at: url, withName: name)
         try createMacroSourceFile(name: name, atPath: path)
     }
+    
+    public func runExternalScriptIfNeeded(at url: URL) throws {
+        guard fileManager.fileExists(atPath: url.path) else { return }
```

Comment:
I'm not sure I like that this function does not notify the user when the script file does not exist. Maybe we should throw an error here and show it in the UI for the user to be aware of the issue? Otherwise things might be broken and the user might not know about it.

Reply:
I just saw that the `shell.executeScript` function already throws an error if it fails to execute the file, so let's remove this guard here and rely on the `Shell` object to communicate that error.